Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -170,6 +170,12 @@ s32 WiiSocket::CloseFd()
     ReturnValue = WiiSockMan::GetNetErrorCode(EITHER(WSAENOTSOCK, EBADF), "CloseFd", false);
   }
   fd = -1;
+  auto it = pending_sockops.begin();
+  while (it != pending_sockops.end())
```

Comment:
```cpp
for (auto it = pending_sockops.begin(); it != pending_sockops.end(); )
```

(I wouldn't move `it = pending_sockops.erase(it);` though, as it's less readable IMO)